README.md

```diff
@@ -131,25 +123,25 @@ or by using the toolbar.
 - **Temperature sensor, Light sensor, acceleration:** click on the corresponding button in the toolbar and change the value using the slider or the input box attached to it.
 - **Shake detection:** go to the motion sensor section in the toolbar and click on the shake button.
 
-### 7. Debug your project on the simulator
+#### 7. Debug your project on the simulator
 
 1. Add breakpoints in your code
 2. Press F5 to enter the debugging mode, and you can start debugging line by line!
 
-## Commands
+### Commands
 
 Device Simulator Express provides several commands in the Command Palette (F1 or Ctrl + Shift + P/ Cmd + Shift + P for Mac OS) for working with \*.py files:
 
-- `Device Simulator Express: New File`: Opens an unsaved .py file with template code, also opens the simulator.
-- `Device Simulator Express: Open Simulator`: Opens the simulator in the webView
-- `Device Simulator Express: Run on Simulator`: Runs python code on the simulator
-- `Device Simulator Express: Deploy to Device`: Copies & Pastes the code.py or main.py file to CIRCUITPY drive if detected a CPX is plugged in
-- `Device Simulator Express: Open Serial Monitor`: Opens the serial monitor in the integrated output window.
-- `Device Simulator Express: Close Serial Monitor`: Stops the serial monitor and releases the serial port.
-- `Device Simulator Express: Change Baud Rate`: Changes the baud rate of the selected serial port. For Adafruit CPX, the default baud rate is 115200.
-- `Device Simulator Express: Select Serial Port`: Changes the current serial port.
+- `Device Simulator Express: [Circuit Playground Express] New File`: Opens an unsaved .py file with template code, also opens the simulator.
+- `Device Simulator Express: [Circuit Playground Express] Open Simulator`: Opens the simulator in the webView
+- `Device Simulator Express: [Circuit Playground Express] Run on Simulator`: Runs python code on the simulator
+- `Device Simulator Express: [Circuit Playground Express] Deploy to Device`: Copies & Pastes the code.py or main.py file to CIRCUITPY drive if detected a CPX is plugged in
+- `Device Simulator Express: [Circuit Playground Express] Open Serial Monitor`: Opens the serial monitor in the integrated output window.
+- `Device Simulator Express: [Circuit Playground Express] Close Serial Monitor`: Stops the serial monitor and releases the serial port.
+- `Device Simulator Express: [Circuit Playground Express] Change Baud Rate`: Changes the baud rate of the selected serial port. For Adafruit CPX, the default baud rate is 115200.
+- `SeleSelect Serial Port`: Changes the current serial port.

```

Comment:
This should probably be `Device Simulator Express: [Circuit Playground Express] Select Serial Port`.